src/main/java/com/ufcg/es5/BackendComplexoEsportivoUFCG/dto/court/CourtUpdateDto.java

```diff
@@ -0,0 +1,19 @@
+package com.ufcg.es5.BackendComplexoEsportivoUFCG.dto.court;
+
+import com.ufcg.es5.BackendComplexoEsportivoUFCG.dto.court.enums.CourtAvailabilityStatusEnum;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.List;
+
+public record CourtUpdateDto(
+        @NotBlank(message = "name cannot be empty")
```

Comment:
se for colocar mensagem nas validações precisa seguir o padrão pra todos os dtos, acho que não é necessário